case-study/sqls/working-hour-distribution/image.svg

```diff
@@ -0,0 +1,230 @@
+<svg
```

Comment:
You don't need to copy the image to case study folder, you can reuse this image in url.

Reply:
I got it!